app/controllers/admin/organizations_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Admin
   class OrganizationsController < BaseController
     def show
-      @organization = Organization.find(params[:id])
+      @organization = Organization.find(current_user.organization_id)
```

Comment:
This should be `@organization = current_user.organization`, there's no need to manually run the lookup.

Also, it looks like the ID from the URL is no longer being used, we should restructure the routes to get rid of it. I think replacing `resources :organizations` with `resource :organization` should work.